machines/backupAndRestore/backup.ts

```diff
@@ -271,24 +326,16 @@ export const backupMachine = model.createMachine(
             entry: ['sendDataBackupFailureEvent', 'cleanupFiles'],
           },
         },
-        on: {
-          DISMISS: {
-            target: 'init',
-          },
-          DISMISS_SHOW_BACKUP_IN_PROGRESS: {
-            actions: 'unsetShowBackupInProgress',
-          },
-        },
       },
     },
   },
   {
     actions: {
-      unsetIsLoading: model.assign({
-        isLoading: false,
+      unsetIsLoadingBackupDetails: model.assign({
```

Comment:
(doubt) This action manages the green/red status bar on the top part of the app which doesn't show up on silent backups, am I right here? I was wondering why this isn't being called on silentFailures due to lack of internet.

Reply:
No, this is handling fetching of last backup details